tools/data/super-resolution/div2k/preprocess_div2k_dataset.py

```diff
@@ -22,6 +22,7 @@ def main_extract_subimages(args):
             A higher value means a smaller size and longer compression time.
             Use 0 for faster CPU decompression. Default: 3, same in cv2.
 
+        scale (List[int]): scale factor
```

Comment:
scale (list[int]): The downsampling factors corresponding to the LR folders you want to process. Default: [2, 3, 4].